software/src/enso/pipe.cpp

```diff
@@ -199,6 +198,18 @@ RxPipe* Device::AllocateRxPipe(bool fallback) noexcept {
   return pipe;
 }
 
+int Device::GetNbFallbackQueues() noexcept {
+  return get_nb_fallback_queues(&notification_buf_pair_);
+}
+
+int Device::SetRrStatus(bool round_robin) noexcept {
+  return set_round_robin_status(&notification_buf_pair_, round_robin);
+}
+
+bool Device::GetRrStatus() noexcept {
```

Comment:
Can you rename it to `GetRoundRobinStatus` to make it consistent with `EnableRoundRobin` and `DisableRoundRobin`?